tripy/tripy/flat_ir/ops/constant.py

```diff
@@ -31,10 +32,11 @@
 @dataclass(repr=False)
 class ConstantOp(BaseFlatIROp):
 
-    data: Array
+    data: Union[Array, Sequence]
```

Comment:
Do we need to be able to give it a shape?

Reply:
Not really, any `shape` argument will already be transformed into a tensor before passing it to `ConstantOp`.